src/integration/control/configureIndex.test.ts

```diff
@@ -136,17 +136,52 @@ describe('configure index', () => {
       });
     });
 
+    test('Add index tag to a serverless index', async () => {
+      const description = await pinecone.describeIndex(serverlessIndexName);
+      expect(description.tags).toEqual({
+        project: 'pinecone-integration-tests',
+      });
+
+      await pinecone.configureIndex(serverlessIndexName, {
+        tags: { testTag: 'testValue' },
+      });
+      const description2 = await pinecone.describeIndex(serverlessIndexName);
+      expect(description2.tags).toEqual({
+        project: 'pinecone-integration-tests',
+        testTag: 'testValue',
+      });
+    });
+
     test('Remove index tag from serverless index', async () => {
       const description = await pinecone.describeIndex(serverlessIndexName);
       expect(description.tags).toEqual({
         project: 'pinecone-integration-tests',
+        testTag: 'testValue',
       });
 
       await pinecone.configureIndex(serverlessIndexName, {
         tags: { project: '' },
       });
       const description2 = await pinecone.describeIndex(serverlessIndexName);
-      expect(description2.tags).toBeUndefined();
+      if (description2.tags != null) {
+        expect(description2.tags['project']).toBeUndefined();
```

Comment:
This is interesting. I'm sure this has already been discussed, but I'd expect to have to explicitly pass `null` for this to happen and for empty strings to be the case, which is how a few other SDKs we use would do it.

Reply:
So the tag itself (the key value) is still present, but the value can be an empty string. If it's an empty string, then it's undefined. Does that make sense?